src/main/java/org/osiam/resources/exception/RestExceptionHandler.java

```diff
@@ -32,18 +32,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
 @ControllerAdvice({
         "org.osiam.auth.oauth_client",
         "org.osiam.metrics.controller",
         "org.osiam.resources.controller"
 })
-public class RestExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
Why don't we want to use this as a base class anymore. Seems quite helpful to me:

> A convenient base class for `@ControllerAdvice` classes that wish to provide centralized exception handling across all `@RequestMapping` methods through `@ExceptionHandler` methods.
> This base class provides an `@ExceptionHandler` method for handling standard Spring MVC exceptions that returns a `ResponseEntity` to be written with message converters.